plugins/Marketplace/resources/qml/Marketplace.qml

```diff
@@ -8,11 +8,13 @@ import QtQuick.Window 2.2
 
 import UM 1.2 as UM
 import Cura 1.6 as Cura
+import Marketplace 1.0 as Marketplace
 
 Window
 {
     id: marketplaceDialog
     property variant catalog: UM.I18nCatalog { name: "cura" }
+    property variant manager: Marketplace.Manager { }
```

Comment:
Now that I read this, why not just set the properties that you need from the manager on the marketplace directly?

Reply:
The (Restar)Manager was intended to also house the check for updates API queries. since we will need that functionality again when the old Toolbox is removed. The red dot we're seeing atm is still the old Toolbox checking for updates.

Gonna give a good thought about this now we're doing the overhaul